policies/releasestrat.html

```diff
@@ -64,15 +50,18 @@ <h5>
 	  and we will specify one at least every four years. Non-LTS
 	  releases will be supported for at least two years.</p>
 
-	  <p>As implied by the above paragraphs, during the final year
+	  <p>During the final year
 	  of support, we do not commit to anything other than security
-	  fixes. Before that, bug and security fixes will be applied
+	  fixes. Before then, bug and security fixes will be applied
 	  as appropriate.</p>
 
 	  <p>The next version of OpenSSL will be 1.1.1. This is currently in
 	  development and has a primary focus of implementing TLSv1.3. The
 	  RFC for TLSv1.3 has not yet been published by the IETF. OpenSSL 1.1.1
-	  will not have its final release until that has happened.</p>
+	  will not have its final release until that has happened;
+          we want to have at least one beta release after TLS 1.3 is
+          officially published as an RFC. We expect 1.1.1 will be our next
+          LTS release.</p>
```

Comment:
The vote that backs this up is still open according to the records, so technically we shouldn't change this until it ends.

The word "expect" in the final sentence adds some element of doubt. The vote is quite clear and does not have that doubt: "The next LTS release will be 1.1.1..."